Mathlib/SetTheory/ZFC/Ordinal.lean

```diff
@@ -49,38 +51,44 @@ theorem isTransitive_iff_mem_trans : z.IsTransitive ↔ ∀ {x y : ZFSet}, x ∈
 alias ⟨IsTransitive.mem_trans, _⟩ := isTransitive_iff_mem_trans
 
 protected theorem IsTransitive.inter (hx : x.IsTransitive) (hy : y.IsTransitive) :
-    (x ∩ y).IsTransitive := fun z hz w hw => by
```

Comment:
These changes really aren't necessary! Both the old and new are valid style (I personally prefer the old version, but it's a matter of taste)